guestmetric/guestmetric_linux.go

```diff
@@ -250,7 +250,7 @@ func (c *Collector) CollectDisk() (GuestMetric, error) {
 				return nil, err
 			}
 			blocksize := 512
-			if bs, err := readSysfs(fmt.Sprintf("/sys/block/%s/queue/physical_block_size", p)); err == nil {
```

Comment:
would you post a screenshot that the current blocksize is incorrect? @ydirson 
Or it needs to some verification from our side. @xihuan-citrix

Reply:
My reading is based on linux 5.10 domains, both in Mageia 8 over Xen 4.13 (xcp-ng) and Debian 11 over Xen 4.14 (QubesOS), where there is no dir entry for the partitions directly in `/sys/block/`:
```
$ ls -dl /sys/block/xvda*
lrwxrwxrwx 1 root root 0 Jul  7 09:48 /sys/block/xvda -> ../devices/vbd-51712/block/xvda
$ ls -dl /sys/block/xvda/xvda*
drwxr-xr-x 5 root root 0 Jul  7 09:48 /sys/block/xvda/xvda1
drwxr-xr-x 5 root root 0 Jul  7 09:48 /sys/block/xvda/xvda2
drwxr-xr-x 5 root root 0 Jul  7 09:48 /sys/block/xvda/xvda3
$ ls -dl /sys/block/xvda/queue/physical_block_size 
-r--r--r-- 1 root root 4096 Jul  7 09:50 /sys/block/xvda/queue/physical_block_size
$ ls -dl /sys/block/xvda1/queue/physical_block_size 
ls: cannot access '/sys/block/xvda1/queue/physical_block_size': No such file or directory
```
If there are kernels which have `queue/physical_block_size` accessible from `/sys/block/xvda1/` and not `/sys/block/xvda/` it could make sense to check both, though, but my guess was that it was just a typo.

Reply:
Hi @minli1 , I'v verified all the guests, I think the modification is correct.